aii-ks/src/main/perl/ks.pm

```diff
@@ -985,7 +985,7 @@ sub ksinstall_rpm
         $disabled = $config->getElement(DISABLED_REPOS)->getTree();
     }
     my $packager = $version >= ANACONDA_VERSION_EL_8_0 ? "dnf" : "yum";
-    my $cmd = "$packager -c /tmp/aii/yum/yum.conf -y install ";
+    my $cmd = "$packager -c /tmp/aii/yum/yum.conf -y -x 'kernel*debug*' install ";
```

Comment:
But... Why? It should be `spma`'s job to control what gets installed. Why do you need to add such custom logic here? And why unconditionally? What happens if someone _wants_ to have kernel debuginfo installed?

Reply:
this is now handled differently (and customisable)